opentracing/src/main/java/opentracing/Span.java

```diff
@@ -62,29 +60,12 @@
   Span setTraceAttribute(String key, String value);
 
   /**
-   * {@code message} is a format string and can refer to fields in the payload by path, like so:
-   *
-   * <pre>{@code
-   *
-   * span.info("first transaction is worth ${transactions[0].amount} ${transactions[0].currency}",
-   *     ImmutableMap.of(
-   *       "transactions", asList(
-   *         Transaction.builder().amount(10).currency("USD").build(),
-   *         Transaction.builder().amount(11).currency("USD").build(),
-   *       )
-   *     )
-   * );
-   * }</pre>
-   *
-   * @param message {@link Formatter format string} that can refer to fields in the args payload.
-   * @param args arbitrary payload
-   */
-  // See https://github.com/opentracing/opentracing.github.io/issues/30 about parameterization
-  Span info(String message, Object... args);
-
-  /** Same as {@link #info}, but for warnings. */
-  Span warning(String message, Object... args);
+   * Add a timestamped annotation, with an optional payload.
+   **/
+  Span log(String message, /* @Nullable */ Object payload);
```

Comment:
TL;DR: what you have here is ok with me if you feel strongly. If it's all the same, I would prefer `logEvent` or `event` for the event-logging version of `log`. LGTM regardless.